CONTRIBUTING.md

```diff
@@ -13,7 +13,10 @@ If you need to make asset changes:
 * `composer install`
 * `npm install`
 * Add the missing values from `.env` to a `.env.local` file
-  * Use `https://vision.googleapis.com/` as the `APP_GOOGLE_CLOUD_VISION_ENDPOINT`, with your own [Cloud Vision API](https://cloud.google.com/vision) key as the `APP_GOOGLE_CLOUD_VISION_KEY`. Google gives you 1,000 free lookups per month.
+  * Create a new Google service account at https://console.cloud.google.com/iam-admin/serviceaccounts Google gives you 1,000 free lookups per month.
+  * Give the service account the *Compute Engine Service Account* role.
```

Comment:
I think there's one more step here, which is to enable the Cloud Vision API. The error I got back from Google gave me a direct link to enable it, but https://console.cloud.google.com/apis/api/vision.googleapis.com/overview should also point to the right place (assuming they're logged in as the right service account)

Reply:
Good point. Done.